client/hooks/useStorageState.ts

```diff
@@ -9,27 +8,19 @@ function useAsyncState<T>(
 ): UseStateHook<T> {
   return React.useReducer(
     (state: [boolean, T | null], action: T | null = null) => [false, action],
-    initialValue
+    initialValue,
   ) as UseStateHook<T>;
 }
 
 export async function setStorageItemAsync(key: string, value: string | null) {
-  if (Platform.OS === 'web') {
-    try {
-      if (value === null) {
-        localStorage.removeItem(key);
-      } else {
-        localStorage.setItem(key, value);
-      }
-    } catch (e) {
-      console.error('Local storage is unavailable:', e);
-    }
-  } else {
+  try {
     if (value == null) {
-      await SecureStore.deleteItemAsync(key);
```

Comment:
We should bring this part back and use SecureStore on native, and keep what you added with AsyncStorage for web

Reply:
Okay that works I thought because we already used it like that then that should work just fine